DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: tidyplus
 Title: Additional 'tidyverse' Functions
-Version: 0.1.0
+Version: 0.1.0.9000
```

Comment:
```suggestion
Version: 0.1.0.9001
```

Reply:
This was a test to practice the commit suggestion thing and thus can be rejected.